src/browser/components/HoveringURL.jsx

```diff
@@ -2,6 +2,10 @@ const React = require('react');
 const PropTypes = require('prop-types');
 
 function HoveringURL(props) {
+  if (props.targetURL.startsWith(props.currentTeamURL)) {
```

Comment:
When mouse cursor moves between internal links and external links, hovering animation is not played because `<CSSTransition>` still exists even if the URL is hidden. So the condition should be examined at MainPage.jsx.

https://github.com/mattermost/desktop/blob/d1b70863b53a2485bdc5d55f9bc2092782bed5bc/src/browser/components/MainPage.jsx#L367-L368